workflow/controller/workflowpod.go

```diff
@@ -535,7 +549,7 @@ func (woc *wfOperationCtx) newWaitContainer(tmpl *wfv1.Template) *apiv1.Containe
 			// in order to SIGTERM/SIGKILL the pid
 			ctr.SecurityContext.Privileged = pointer.BoolPtr(true)
 		}
-	case "", common.ContainerRuntimeExecutorDocker:
+	case common.ContainerRuntimeExecutorDocker:
```

Comment:
this is an existing bug, I think we need a fix for this from the author
Created #7118

Reply:
should I revert that line back to the original then?

Reply:
I think we should get that fixed first

Reply:
of course, you could submit that fix too :)